src/Services/DataService/DataService.test.ts

```diff
@@ -871,34 +871,38 @@ describe("DataService", () => {
           });
     });
 
-    it("patchRecordEpic should call transform before serialize", () => {
+    it("patchRecordEpic should call transformPatch before serialize", () => {
       const onSuccess = spy();
-      const expectedResult = { id: "123", fullText: "zella puppy transform" };
+      const expectedResult = JSON.stringify([{ op: "replace", field: "/fullText", value: lorem.slug() }]);
       const patchRecordAction = fakeService.actions.patchRecord(expectedResult, { onSuccess });
-      const transformStub = stub(fakeService.mapper, "transform");
+      const transformStub = stub(fakeService.mapper, "transformPatch");
 
       stub(fakeService.serializer, "serialize").returns(expectedResult);
 
-      fakeService.patchRecordEpic(ActionsObservable.of(patchRecordAction), store)
-        .subscribe(noop, noop,
-          () => {
+      return new Promise(resolve => {
+        fakeService.patchRecordEpic(ActionsObservable.of(patchRecordAction), store)
+          .subscribe(noop, noop, () => {
             expect(transformStub.firstCall.args[0]).to.equal(expectedResult);
+            resolve();
           });
+      });
     });
 
-    it("patchRecordEpic should serialize the result from transform", () => {
+    it("patchRecordEpic should serialize the result from transformPatch", () => {
       const onSuccess = spy();
-      const expectedResult = { id: "123", fullText: "zella puppy serialize transform" };
+      const expectedResult = [{ op: "replace", path: "/fullText", value: lorem.slug() }];
       const patchRecordAction = fakeService.actions.patchRecord(expectedResult, { onSuccess });
 
-      stub(fakeService.mapper, "transform").returns(expectedResult);
+      stub(fakeService.mapper, "transformPatch").returns(expectedResult);
       const serialStub = stub(fakeService.serializer, "serialize");
 
-      fakeService.patchRecordEpic(ActionsObservable.of(patchRecordAction), store)
-        .subscribe(noop, noop,
-          () => {
+      return new Promise(resolve => {
```

Comment:
async assertions need promises because the test has already finished and been marked as a pass by the time it fails.  The other tests in this file that use async assertions should also get this treatment.